listener/src/main.rs

```diff
@@ -13,7 +15,11 @@ use std::sync::Arc;
 use std::time::Duration;
 
 use graph::components::forward;
-use graph::prelude::*;
+use graph::prelude::{
+    SubgraphRegistrar as SubgraphRegistrarTrait,
+    SubgraphAssignmentProvider as SubgraphAssignmentProviderTrait,
+    *
```

Comment:
Looks like we only need `SubgraphName` and `SubgraphDeploymentId` in addition to those already explicitly imported. Do we expect to use other graph preludes later?

Reply:
Sure, I can make dependencies explicit.